.travis.yml

```diff
@@ -1,25 +1,35 @@
 language: java
 jdk:
   - oraclejdk8
-install:
-  - mvn --settings .travis/settings.xml install -DskipTests=true -Dgpg.skip -Dmaven.javadoc.skip=true -B -V
+  - openjdk8
+  - openjdk11
+matrix:
+  allow_failures:
+    - jdk: openjdk11
+install: true
 before_install:
   - if [ ! -z "$GPG_SECRET_KEYS" ]; then echo $GPG_SECRET_KEYS | base64 --decode | $GPG_EXECUTABLE --import; fi
   - if [ ! -z "$GPG_OWNERTRUST" ]; then echo $GPG_OWNERTRUST | base64 --decode | $GPG_EXECUTABLE --import-ownertrust; fi
-deploy:
-  -
-    provider: script
-    script: .travis/deploy.sh
-    skip_cleanup: true
-    on:
-      repo: rapid7/smbj-rpc
-      branch: master
-      jdk: oraclejdk8
-  -
-    provider: script
-    script: .travis/deploy.sh
-    skip_cleanup: true
-    on:
-      repo: rapid7/smbj-rpc
-      tags: true
+script:
+  - mvn test -Dgpg.skip -Dmaven.javadoc.skip=true -B -V -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```

Comment:
The `-Dorg.slf4j.simpleLogger.log...` part is what hides all the `Downloading ...` log messages from maven, which makes the logs a lot quicker to skim through.

Hindsight: the `-Dgpg.skip` and `-Dmaven.javadoc.skip=true` flags shouldn't be necessary anymore with the release profile setup.